Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM alpine:latest as layer-copy
```

Comment:
nit: can we remove the alias as layer-copy. We are not using the multi-stage build as described [here](https://aws.amazon.com/blogs/compute/working-with-lambda-layers-and-extensions-in-container-images/#:~:text=The%20Dockerfile%20for%20this%20automated%20approach%2C%20to%20add%20two%20layers%20into%20a%20single%20image%2C%20includes%20the%20following%20lines%20to%20copy%20the%20Lambda%20layer%20contents.)